### ISSUE TITLE
Test "prGetSource" raise "index must be an Integer"  error 

### ISSUE THREAD

**Issue by oliviersurf** [escuta/mosca#56]
Hello,

First of all, congratulations for your work !

I have the following error "index must be an Integer" when i use this method (but also any other method) :

```
~moscaInstance.getSCBus(1);
~moscaInstance.setCartesian(1, 0, 0,0);
...
```

I think it comes from the "prGetSource" method, the condition should be "if not".

```

	prGetSource
	{ | sourceNum |

		if (sourceNum.isInteger)
		{
			^Error("index must be an Integer").throw;
		} {
			^sources.get[(sourceNum.clip(1, sources.get.size)) - 1];
		};
	}
```

What do you think ?
Thanks !

**Comment by thibaudk** [comment]
Hi,

yes absolutely !
Silly mistake on my part.

I'll add the fix in the next tag.

Thanks for reporting !

**Comment by thibaudk** [comment]
fixed with c223f41
